_component_design/vertical-stepper.md

```diff
@@ -4,31 +4,256 @@ parent: Content Areas
 
 <section class="static-section" markdown="1">
 
-## Step 2
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Stepper counter
+
+The stepper counter displays the sequential number of a step as well as the status of the step. 
 
-{% figure [caption:""] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/stepper-2.svg){:width="100%"}
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [caption:"Stepper counter states and specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/vert-stepper/vert-stepper-counter.png){: width="580px"}
 {% endfigure %}
 
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Clickable area
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [caption:"Stepper expand / collapse clickable areas"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/vert-stepper/vert-stepper-clickable.png){: width="903px"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+### Loading state
+
+{% figure [caption:"Stepper loading state"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/vert-stepper/vert-stepper-loading.png){: width="903px"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+## Variances
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Submission controls
+
+As the stepper pattern isolates each sequential step within a multistep process, a variance in the buttons displayed within each step will become apparent to the user. Dependent on which sequential step is displayed, the following buttons may be displayed.
+
+**First step:** Displays a primary button to move to the next step and tertiary button to cancel the form. 
+
+**Intermediate step(s):** Displays primary and secondary buttons to move between steps and a tertiary button to cancel the form.
+
+**Final step:** Mimics the intermediate step, however the primary button is now used to complete the form and labeled according; For example, ‘Submit’ or ‘Confirm’.
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [caption:"Button displays based on currently displayed step."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/vert-stepper/vert-stepper-buttons.png){: width="580px"}
```

Comment:
Just putting a pin in this for future change:

At some point in the near future, we should consider revising our button labels here to something like "Continue", and "Go Back" -- but that can be handled in a fast-follow change.

Reply:
Image updated to "Continue", and "Go Back"